app/views/plans/_edit_details.html.erb

```diff
@@ -1,288 +1,23 @@
-<% project_title_tooltip =  _('If applying for funding, state the name exactly as in the grant proposal.') %>
-<% project_abstract_tooltip = _("Briefly summarise your research project to help others understand the purposes for which the data are being collected or created.") %>
-<% id_tooltip = _('A pertinent ID as determined by the funder and/or organisation.') %>
+<%# locals: plan %>
 
-<div class="row">
+<%= form_for plan, html: { method: :put,
+                            class: 'form-horizontal edit_plan' } do |f| %>
+  <div class="row">
     <div class="col-md-8">
-    <%= form_for plan, html: {method: :put, class: 'form-horizontal edit_plan' } do |f| %>
-      <div class="form-group"><!-- title -->
-        <div class="col-md-12">
-          <%= f.label(:title, _('Project title'), class: 'control-label') %>
-        </div>
-        <div class="col-md-8">
-          <em class="sr-only"><%= project_title_tooltip %></em>
-          <%= f.text_field(:title, class: "form-control", "aria-required": true,
-            'data-toggle': 'tooltip', spellcheck: true,
-            title: project_title_tooltip) %>
-        </div>
-        <div class="col-md-8">
-          <div class="checkbox">
-            <%= f.hidden_field :visibility %>
-            <%= f.label(:is_test, class: 'control-label') do %>
-              <%= check_box_tag(:is_test, 1, @plan.is_test?, "aria-label": "is_test") %>
-              <%= _('mock project for testing, practice, or educational purposes') %>
-            <% end %>
-          </div>
-        </div>
-      </div>
+      <%= render partial: "plans/project_details",
+                 locals: { form: f, plan: plan } %>
```

Comment:
I did not realize that all of this functionality was in the one view-file.
Thanks for breaking this out into partials